### PR TITLE
Rendering Harvest data on the ember side

### PR DESCRIPTION
**Pull request by jir-f**

### CODE REVIEW

File: app-ember/.gitignore [hunk 1]
```diff
@@ -5,7 +5,7 @@
 /tmp
 
 # dependencies
-/node_modules
```
Comment (HaidenD) [inline app-ember/.gitignore]: xD who did this?

File: app-ember/app/components/overview-harvest.js [hunk 2]
```diff
@@ -1,4 +1,27 @@
 import Ember from 'ember';
 
+var out;
```
Comment (HaidenD) [inline app-ember/app/components/overview-harvest.js]: is the var out and console.log(out) necessary.

File: app-ember/.gitignore (hunk 1 above)
Comment (jir-f) [reply app-ember/.gitignore]: No clue -_-

File: app-ember/package.json [hunk 3]
```diff
@@ -45,6 +45,7 @@
     "ember-fetch": "^3.4.0",
     "ember-freestyle": "^0.6.1",
     "ember-load-initializers": "^1.0.0",
+    "ember-lodash": "^4.17.5",
```
Comment (HaidenD) [inline app-ember/package.json]: Niceee Lodash has a lot of handy features.

File: app-ember/app/components/overview-harvest.js (hunk 2 above)
Comment (jir-f) [reply app-ember/app/components/overview-harvest.js]: Not really 🙄 I forgot it there

**Pull request merged**